src/cclib/parser/data.py

```diff
@@ -70,6 +76,8 @@ class ccData(object):
         scftargets -- targets for convergence of the SCF (array[2])
         scfvalues -- current values for convergence of the SCF (list of arrays[2])
         temperature -- temperature used for Thermochemistry (float, kelvin)
+        theory -- name of the quantum chemistry methodology (string)
+        version -- the version number for quantum chemistry package (string)

```

Comment:
Attributes should be results of the calculations. We definitely don't want to create attributes for specific metadata elements.